app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  def index
+    @comments = Comment.all(post: @post)
```

Comment:
You need to define `@post` in this controller action before you can reference it. I would do that on the prior line.

You would then be able to write out something like `@comments = @post.comments`